.npmrc

```diff
@@ -1 +0,0 @@
-@careevolution:registry=https://registry.npmjs.org
```

Comment:
@reneefromhold can we keep this?  Is there a reason to remove it?

Reply:
Actually, I think you will need to add this back and then regenerate your package-lock.json because I see references to proget, which people outside CE won't be able to hit (so for someone external to CE if they try to use it it won't work)